src/components/view/index.tsx

```diff
@@ -35,6 +35,10 @@ export interface ViewProps extends Omit<RNViewProps, 'style'>, ReanimatedProps,
    * TODO: probably isn't needed
    */
   height?: string | number;
+  /**
+   * Set flexbox gap
+   */
+  gap?: number;
```

Comment:
Adding the type doesn't necessarily means that it'll work.. and if it works the type should be part of RN's View types 
Passing `gap` works for you? (both on iOS and Android)